src/file/mod.rs

```diff
@@ -232,7 +220,7 @@ fn configure_hostname(
     ])
 }
 
-fn get_file_path(parent: Option<&Path>, file_name: &str) -> Result<PathBuf> {
+pub(crate) fn get_file_path(parent: Option<&Path>, file_name: &str) -> Result<PathBuf> {
     let mut file_path = parent
         .context("get_file_path: cannot get parent directory")?
         .to_path_buf();
```

Comment:
Nit/refactor: why do we allow to pass in an option here, if we bail if it is unset anyway?